examples/jupyter-tutorials/API.ipynb

```diff
@@ -11,188 +11,1039 @@
-    "p = m.add_parameters((3,5), init='normal', mean=a, std=b)   # Creates 3x5 matrix initialized with N(a, b)\n",
-    "p = m.add_parameters((5,5), init='identity')                # Creates 5x5 identity matrix\n",
-    "p = m.add_parameters((3,5), init='glorot')                  # Creates 3x5 matrix with glorot init\n",
-    "p = m.add_parameters((3,5), init='he')                      # Creates 3x5 matrix with he init\n",
-    "p = m.add_parameters(np.zeros((3,5)))                       # Creates 3x5 matrix from a numpy array (size is inferred)\n",
-    "p = m.add_parameters((3,5), init=dy.PyInitializer())        # Any parameter initializer\n",
+    "scale, mean, stddev = 1, 0, 1\n",
     "\n",
+    "# Creates 3x5 matrix filled with 0 (or any other float)\n",
+    "p = m.add_parameters((3,5), init=0)\n",
+    "# Creates 3x5 matrix initialized with U([-scale, scale])\n",
+    "p = m.add_parameters((3,5), init='uniform', scale=scale)\n",
+    "# Creates 3x5 matrix initialized with N(mean, stddev)\n",
+    "p = m.add_parameters((3,5), init='normal', mean=mean, std=stddev)\n",
+    "# Creates 5x5 identity matrix\n",
+    "p = m.add_parameters((5,5), init='identity')\n",
+    "# Creates 3x5 matrix with glorot init\n",
+    "p = m.add_parameters((3,5), init='glorot')\n",
+    "p = m.add_parameters((3,5)) # By default, the init = 'glorot'\n",
+    "# Creates 3x5 matrix with he init\n",
+    "p = m.add_parameters((3,5), init='he')\n",
+    "# Creates 3x5 matrix from a numpy array (size is inferred)\n",
+    "p = m.add_parameters((3,5), np.ones((3,5)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create LookupParameters\n",
+    "LookupParameters represents a table of parameters. They are used to embed a set of discrete objects (e.g. word embeddings). They can be sparsely updated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "## ===== Lookup parameters\n",
-    "# Similar to parameters, but are representing a \"lookup table\"\n",
-    "# that maps numbers to vectors.\n",
+    "# Similar to parameters, but are representing a \"lookup table\" that maps numbers to vectors.\n",
     "# These are used for embedding matrices.\n",
-    "# for example, this will have VOCAB_SIZE rows, each of DIM dimensions.\n",
-    "lp  = m.add_lookup_parameters((VOCAB_SIZE, DIM))\n",
-    "\n",
-    "# lookup parameters can be initialized from an existing array, i.e:\n",
-    "# m[\"lookup\"].init_from_array(wv)\n",
-    "\n",
+    "# For example, this will have VOCAB_SIZE rows, each of DIM dimensions.\n",
+    "VOCAB_SIZE = 100\n",
+    "DIM = 10\n",
+    "lp = m.add_lookup_parameters((VOCAB_SIZE, DIM))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Ceate expressions from lookup parameters.\n",
```

Comment:
Ceate -> Create

(There may be other spelling errors, so I'd try to spell-check the text)